docs/guides/pam.rst

```diff
@@ -0,0 +1,54 @@
+Pluggable Authentication Modules / PAM
+######################################
+
+Class :class:`sssd_test_framework.utils.pam.PAMUtils` provides
+an API to manage PAM modules; Currently pam_access and pam_faillock.
+
+pam_access: A module that performs host based access control on a system.
+
+.. code-block:: python
+    :caption: Example PAM Access usage
+     @pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+     def test_example(client: Client, provider: GenericProvider):
+        # Add users
+        provider.user("user-1").add()
+        provider.user("user-1").add()
+
+        # Add rule to permit "user-1" and deny "user-2"
+        client.pam.access.add(["+:user-1:ALL","-:user-2:NONE"])
```

Comment:
I think pam.access.add  is adequate, I'll defer the decision to the second reviewer.

Reply:
Depends. It would be nice to have the API unified in both pam_access and pam_faillock, currently you chose little  different approach in each. Since PAMFaillock also allows to work with the database, using config_ prefix is a good idea. That is:  `config_write/read/add/delete/set`

Reply:
Pavel's explanation is way better than mine. Let's have a unified API